files/healthcheck.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+STACK_NAME=$1
+RESOURCE_NAME=${2:-AutoscalingGroup}
+AWS_REGION=${3:-eu-west-1}
+MAX_RETRIES=${4:-10}
+RETRIES=1
+SLEEP=2
+
+function cfn-signal() {
+  /usr/bin/docker run --rm mbabineau/cfn-bootstrap cfn-signal \
+    -e $? \
+    --stack ${STACK_NAME} \
+    --resource ${RESOURCE_NAME} \
+    --region ${AWS_REGION}
+}
+trap cfn-signal EXIT
+
+sleep 5 # It takes compose a few seconds to create the container
```

Comment:
I want to test it again, just to make sure, but I believe Compose does a weird order of execution when starting containers